### PR TITLE
[11.0][MIG] sale_merge_draft_invoice

### PR DESCRIPTION
**Pull request by rgarnau**
No major changes

Dependencies: 
- [x] #567

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]
:+1:

File: sale_merge_draft_invoice/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+========================
+Sale Merge Draft Invoice
+========================
+
+This module merges new invoices with existing draft ones when creating them.
+
+The merge between invoices is done when the partner and the currency are the
+same.
+
```
Comment (gdgellatly) [inline sale_merge_draft_invoice/README.rst]: I think this section should reference #568 capabilities and Odoo default capabilities.  The way it reads now, it reads like this module groups by partner and currency, when in fact it doesn't, it just considers draft invoices in the grouping. 

Explain the default group of Odoo like you have, then explain that alternative custom groupings can be achieved using that module.

Something like.

Odoo can merge sales orders into a single invoice, grouped by partner and currency, when the sales orders are selected and invoiced together.  This module extends the grouping to also consider any pre-existing draft invoices and merge them.  

If you also need to group on fields other than partner and currency, the OCA module sale_invoice_group_method allows you to specify alternative grouping methods and is fully compatible with this module.

**Review by gdgellatly — APPROVED** [review]
Minor doc comment, rest LGTM.

**Comment by rgarnau** [comment]
@gdgellatly - I really like the way you explain it! Already changed! Thank you!!

**Review by etobella — APPROVED** [review]
LGTM :+1:

**Pull request merged**